api/controllers/budgetController.js

```diff
@@ -0,0 +1,21 @@
+const budgetModel = require('../models/budgetModel'); //mongoose instance
+
+const getBudget = (req, res) => {
+    budgetModel.find({})
+        .populate()
+        .exec((err, post) => {
+            console.log('hit getBudget');
```

Comment:
Reminder to remove console logs when finished with them

Reply:
done! 👍